test/writeable_test.rb

```diff
@@ -26,4 +26,33 @@ class LocationForm < Reform::Form
 
     hash.must_equal("country"=> "Germany")
   end
-end
\ No newline at end of file
+end
+
+# writable option is alias of wiriteable option.
```

Comment:
If I'm being super picky, there's a type in your comment... 😜 

wiriteable => writeable